src/protocol/attribution/credit_capping.rs

```diff
@@ -310,12 +310,11 @@ impl AsRef<str> for Step {
 
 #[cfg(all(test, not(feature = "shuttle")))]
 mod tests {
-    use super::super::tests::generate_shared_input;
     use crate::{
         ff::{Field, Fp32BitPrime},
         protocol::attribution::{
             credit_capping::credit_capping,
-            tests::{BD, H, S, T},
+            tests::{generate_shared_input, BD, H, S, T},
```

Comment:
this is unrelated change: I moved things around here and Clion decided to format it this way. Decided to keep it because PR is small and it is better to import it this way